webapp/src/components/viewHeader.tsx

```diff
@@ -68,6 +69,23 @@ class ViewHeader extends React.Component<Props, State> {
         }
     }
 
+    onExportCsvTrigger(boardTree: BoardTree, intl: IntlShape) {
```

Comment:
the `intl` object is avaiable in `this.props.intl`, I think is better to use it directly instead of pass this as a parmeter.

Reply:
ah, did not notice. Thanks for the tip!
Made changes https://github.com/mattermost/focalboard/pull/178/commits/a590409ae295cd7f1c0ce309ea37ae1c7cca5176 😄